### PR TITLE
Fix dead link

### PR DESCRIPTION
**Pull request by jcrater**
## Before this PR
RFC 002 link is dead.

## After this PR
==COMMIT_MSG==
RFC 002 link works.
==COMMIT_MSG==

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ convenience, but users must assume responsibility for adapting their code when d
 
 ## Usage
 
-The recommended way to use conjure-java is via a build tool like [gradle-conjure](https://github.com/palantir/gradle-conjure). However, if you don't want to use gradle-conjure, there is also an executable which conforms to [RFC 002](https://github.com/palantir/conjure/blob/develop/rfc/002-contract-for-conjure-generators.md),  published on [bintray](https://bintray.com/palantir/releases/conjure-java).
+The recommended way to use conjure-java is via a build tool like [gradle-conjure](https://github.com/palantir/gradle-conjure). However, if you don't want to use gradle-conjure, there is also an executable which conforms to [RFC 002](https://github.com/palantir/conjure/blob/master/docs/rfc/002-contract-for-conjure-generators.md),  published on [bintray](https://bintray.com/palantir/releases/conjure-java).
```
Comment (carterkozak) [inline readme.md]:
```suggestion
The recommended way to use conjure-java is via a build tool like [gradle-conjure](https://github.com/palantir/gradle-conjure). However, if you don't want to use gradle-conjure, there is also an executable which conforms to [RFC 002](docs/rfc/002-contract-for-conjure-generators.md),  published on [bintray](https://bintray.com/palantir/releases/conjure-java).
```

I think we can use a relative link to be consistent with the current branch, whichever that may be.

Comment (carterkozak) [reply readme.md]: Also, thanks for catching this!

Comment (dansanduleac) [reply readme.md]: @carterkozak it's in a different repo. I think we should instead link to the rendered docs

Comment (dansanduleac) [reply readme.md]:
```suggestion
The recommended way to use conjure-java is via a build tool like [gradle-conjure](https://github.com/palantir/gradle-conjure). However, if you don't want to use gradle-conjure, there is also an executable which conforms to [RFC 002](https://palantir.github.io/conjure/#/docs/rfc/002-contract-for-conjure-generators),  published on [bintray](https://bintray.com/palantir/releases/conjure-java).
```

**Comment by dansanduleac** [comment]
@jcrater we can't modify your branch because the PR doesn't have 'allow maintainers to edit' enabled, but if you apply my suggested change I can approve.

**Comment by jcrater** [comment]
@dansanduleac I see `Allow edits from maintainers` as checked. Not sure if an admin checked that, but I committed your change either way. Thanks.

**Comment by dansanduleac** [comment]
👍

**Pull request merged**